project-base/src/DataFixtures/Demo/ImageDataFixture.php

```diff
@@ -254,21 +254,28 @@ private function processSliderItemsImages()
      */
     private function saveImageIntoDb(int $entityId, string $entityName, int $imageId)
     {
-        $query = $this->em->createNativeQuery(
+        $this->em->getConnection()->executeStatement(
             'INSERT INTO images (id, entity_name, entity_id, type, extension, position, modified_at)
             VALUES (:id, :entity_name, :entity_id, :type, :extension, :position, :modified_at)',
-            new ResultSetMapping()
+            [
+                'id' => $imageId,
+                'entity_name' => $entityName,
+                'entity_id' => $entityId,
+                'type' => null,
+                'extension' => self::IMAGE_TYPE,
+                'position' => null,
+                'modified_at' => new DateTimeImmutable('2015-04-16 11:36:06'),
```

Comment:
Not necessary to use parameters.

Reply:
removed unnecessary in 2bf489594. 
extension kept on purpose as it's more readable than escaped apostrophes in sql string.